src/ngraph/runtime/cpu/builder/add.cpp

```diff
@@ -40,7 +40,7 @@ namespace ngraph
 
                     auto& mkldnn_emitter = external_function->get_mkldnn_emitter();
                     auto sum_pd = mkldnn_emitter->get_elementwise_add_desc(node);
-                    QUERY_SCRATCHPAD(sum, sum_pd);
+                    size_t s_size = QUERY_SCRATCHPAD(sum, sum_pd);
```

Comment:
s_* is used for static variables. Rename to something like 'scratchpad_size' or 'tmpbuf_size'